### DISCUSSION TITLE
How to create a new endpoint?

### DISCUSSION THREAD

**Discussion by yaten2302** [anuraghazra/github-readme-stats#3193] [Q&A] [answered]
Hi @qwerty541, as you told in #3183, you told how to create a new gist card (#3064), could you also please tell that how can I create a new endpoint for the new changes that I've made and also how to view those changes locally like on localhost?

**Comment by qwerty541** [comment, marked as answer]
@yaten2302 Pull request #3064 which i mentioned in my previous answer also contains new endpoint addition (file `/api/gist.js`). Every file inside `api` folder is a separate endpoint. To add your new endpoint you should simply add new file in this folder using any other file like example for retrieving query options and handling common errors. To view your changes locally, after adding new endpoint file and running `vercel dev` command to init development environment, you should open the following URL in browser: `localhost:3000/api/{YOUR_ENDPOINT_FILE_NAME}?{QUERY_OPTIONS}`
